pom.xml

```diff
@@ -4,35 +4,43 @@
 	<!-- This project -->
 	<groupId>com.ft.unifiedContentModel</groupId>
 	<artifactId>ft-unifiedContentModel-parent</artifactId>
-	<version>0.41.0</version>
+	<version>1.0.0</version>
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<spring.version>3.1.1.RELEASE</spring.version>
-		<spring.ws.version>2.0.2.RELEASE</spring.ws.version>
-		<servlet-api.version>2.5</servlet-api.version>
-		<java.version>1.6</java.version>
-		<jackson.version>1.9.7</jackson.version>
+		<spring.version>5.3.9</spring.version>
+		<spring.boot.version>2.5.3</spring.boot.version>
+		<spring.ws.version>5.3.9</spring.ws.version>
+		<servlet-api.version>4.0.1</servlet-api.version>
+		<javax.annotation-api.version>1.3.2</javax.annotation-api.version>
+		<java.version>11</java.version>
+		<jackson.version>2.12.4</jackson.version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<joda-time.version>1.6.2</joda-time.version>
-		<guava.version>r09</guava.version>
-		<commons-beanutils.version>1.8.0</commons-beanutils.version>
-		<commons-io.version>2.0.1</commons-io.version>
+		<!--		FIXME migrate to java.time https://www.oracle.com/technical-resources/articles/java/jf14-date-time.html-->
```

Comment:
Will the FIXMEs and TODOs be resolved in follow-up PRs?

Reply:
@mitagg I would like to pospone the migration/remove of this time library to make progress on search API port

Reply:
Agreed